src/options.zig

```diff
@@ -684,6 +684,8 @@ pub const Loader = enum(u8) {
             .napi,
             .sqlite,
             .sqlite_embedded,
+            // TODO: loader for reading bytes and creating module or instance
```

Comment:
changing the default behavior of a loader after implementing is a breaking change. so implementing this todo would be a separate loader. like we could do `with { type: 'wasm' }`. not important now.